pkg/alertmanager/types.go

```diff
@@ -431,7 +431,7 @@ type msTeamsConfig struct {
 
 type msTeamsV2Config struct {
 	SendResolved   *bool             `yaml:"send_resolved,omitempty"`
-	WebhookURL     string            `yaml:"webhook_url"`
+	WebhookURL     string            `yaml:"webhook_url,omitempty"`
```

Comment:
Is this an optional field?

Reply:
Both WebhookURLFile and WebhookURL are optional and should either be used one or another per documentation https://prometheus.io/docs/alerting/latest/configuration/#msteamsv2_config

Reply:
Was checking https://github.com/prometheus/alertmanager/pull/3555/files, added omitempty as well to WebhookURL